### PR TITLE
docs: Сorrects a mistake

### PR DESCRIPTION
**Pull request by mephistophelius**

### CODE REVIEW

**Comment by stylesam** [comment]
Hey, why my build fail ?

**Comment by saulotoledo** [comment]
@jotamorais

File: README.md [hunk 1]
```diff
@@ -600,8 +600,8 @@ export class User {
 
 ```typescript
 import {classToPlain} from "class-transformer";
-let user1 = classToPlain(user, { groups: ["user"] }); // will contain id, name, email and password
-let user2 = classToPlain(user, { groups: ["admin"] }); // will contain id, name and email
+let user1 = classToPlain(user, { groups: ["user"] }); // will contain id, name and email
+let user2 = classToPlain(user, { groups: ["admin"] }); // will contain id, name, email and password
```
Comment (saulotoledo) [inline README.md:604]: This is not right. According to the example, the property `password` is in the group `user`. In that case, the original documentation is correct.

In fact, there is an error in the documentation, but it is not covered in this PR: after using `@Exclude()` in the class,, the properties id and name are removed.

**Comment by saulotoledo** [comment]
@mephistophelius, as I commented on your PR, the issue with the example was due to the `@Exclude()`, fixed at https://github.com/typestack/class-transformer/pull/371. I am closing this PR, but please feel free to argue if you still believe something else should be modified in that documentation.

**Pull request closed without merging**